src/schemathesis/commands.py

```diff
@@ -0,0 +1,56 @@
+import re
+from typing import Iterable
+
+import click
+
+from . import runner
+
+CONTEXT_SETTINGS = dict(help_option_names=["-h", "--help"])
+
+default_checks = tuple(check.__name__ for check in runner.default_checks)
+
+
+@click.group(context_settings=CONTEXT_SETTINGS)
+@click.version_option()
+def main() -> None:
+    """Command line tool for testing your web application built with Open API / Swagger specifications."""
+
+
+@main.command(short_help="Perform schemathesis test.")
+@click.argument("schema", type=str)
+@click.option("--url", help="URL address of the API, required for SCHEMA if specified by file.", type=str)
+@click.option(
+    "--checks",
+    "-c",
+    multiple=True,
+    help="List of checks to run.",
+    type=click.Choice(default_checks),
+    default=default_checks,
+)
+def run(schema: str, url: str = "", checks: Iterable[str] = default_checks) -> None:
+    """Perform schemathesis test against an API specified by SCHEMA.
+
+    SCHEMA must be a valid URL or a file path.
+    """
+    is_file_path = re.search(r"^https?://", schema) is None
```

Comment:
I am not currently sure about a file-based schema in CLI and would rather postpone it a bit (in a separate PR probably) to start with a minimal CLI. What do you think?

Reply:
I implemented it because I wanted to test it locally in another app which didn't have the `swagger.json` URL anywhere. It seemed very useful for testing, I can imagine more use-cases (like testing new schema against production or something)...

But I can remove it from this PR definitely, but since it's implemented, should I? Or do you see a big issue with this?

Reply:
Local schemas could be tested with `file://` schema, so there will be two options to do the same thing. I need to think more about it and to avoid waiting I wanted to start with the simpler implementation to unblock implementing custom options, etc. I absolutely agree that this feature is useful, I only not sure what would be the best interface for it yet

Reply:
OK, makes sense. I will remove the option for now (but keep it in another branch if we get back to it).

That said, I personally don't like having `file://`` in front, as the autocomplete feature in bash/zsh doesn't work for me that way. We could make it more explicit by having an explicit option like `-f` or something. But I also like simple APIs which don't require that and are kind of smart (I think the differentiation between URL and file path is straightforward here if we will require `http(s)` in front of the URL).